tests/test_modeling_common.py

```diff
@@ -590,7 +589,7 @@ def test_inputs_embeds(self):
                 inputs_dict["decoder_inputs_embeds"] = wte(decoder_input_ids)
 
             with torch.no_grad():
-                outputs = model(**inputs_dict)
+                model(**inputs_dict)
```

Comment:
I kept this, and all similar cases, because I assume we're testing that the model runs without crashing.